stable/chartmuseum/templates/deployment.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ include "chartmuseum.fullname" . }}
+  annotations:
+{{ toYaml .Values.deployment.annotations | indent 4 }}
+  labels:
+{{ include "chartmuseum.labels.standard" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+{{ toYaml .Values.strategy | indent 4 }}
+  revisionHistoryLimit: 10
+  template:
+    metadata:
+      name: {{ include "chartmuseum.fullname" . }}
+      annotations:
+{{ toYaml .Values.replica.annotations | indent 8 }}
+      labels:
+        app: {{ template "chartmuseum.name" . }}
+        release: {{ .Release.Name | quote }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+{{- range $name, $value := .Values.env.open }}
+{{- if not (empty $value) }}
+        - name: {{ $name | quote }}
+          value: {{ $value | quote }}
+{{- end }}
+{{- end }}
+{{- $secret_name := include "chartmuseum.fullname" . }}
+{{- range $name, $value := .Values.env.secret }}
+{{- if not ( empty $value) }}
+        - name: {{ $name | quote }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ $secret_name }}
+              key: {{ $name | quote }}
+{{- end }}
+{{- end }}
+        args:
+        - --port={{ .Values.service.internalPort }}
+{{- if eq .Values.env.open.STORAGE "local" }}
+        - --storage-local-rootdir=/storage
+{{- end }}
+        ports:
+        - containerPort: {{ .Values.service.internalPort }}
```

Comment:
Does it make sense to have the container port configurable? Just wondering. Also, you could give it a name and reference it by its name from service and probes.